src/components/SelectionList/BaseSelectionList.tsx

```diff
@@ -427,7 +427,8 @@ function BaseSelectionList<TItem extends ListItem>(
                     shouldPreventDefaultFocusOnSelectRow={shouldPreventDefaultFocusOnSelectRow}
                     // We're already handling the Enter key press in the useKeyboardShortcut hook, so we don't want the list item to submit the form
                     shouldPreventEnterKeySubmit
-                    rightHandSideComponent={rightHandSideComponent}
+                    // Change this because of lint
```

Comment:
Can we leave a better comment here? I'm not sure what this means.

Reply:
Yeah, makes sense. But since we're touching this code, let's still make this comment better while we're here. To leave the code in a better place than when we found it.